pluma/pluma-view.c

```diff
@@ -825,9 +825,9 @@ pluma_override_font (const gchar          *item,
 				  "changed::" "font-name",
 				  G_CALLBACK (system_font_changed_cb), NULL);
 
-		gtk_style_context_add_provider_for_screen (gtk_widget_get_screen (widget),
-							   GTK_STYLE_PROVIDER (provider),
-							   GTK_STYLE_PROVIDER_PRIORITY_APPLICATION);
+		gtk_style_context_add_provider (gtk_widget_get_style_context (widget),
+						GTK_STYLE_PROVIDER (provider),
+						GTK_STYLE_PROVIDER_PRIORITY_APPLICATION);
```

Comment:
It seems this change is a regression, it comes back the issue https://github.com/mate-desktop/pluma/issues/234

Reply:
any steps on how to test? I tested it with this PR and can not reproduce the issue described.

Reply:
![2020-05-05_23-11](https://user-images.githubusercontent.com/7734191/81116635-03975500-8f26-11ea-9029-e7990eb60017.png)
![2020-05-05_23-12](https://user-images.githubusercontent.com/7734191/81116636-04c88200-8f26-11ea-86c0-e6394ae1de9e.png)

Expected behavior (without PR):

![2020-05-05_23-06](https://user-images.githubusercontent.com/7734191/81116686-1ad64280-8f26-11ea-8542-c7c9ff990701.png)

Actual behavior (with PR):

![2020-05-03_04-05](https://user-images.githubusercontent.com/7734191/81116708-2295e700-8f26-11ea-957d-75df98ee270f.png)

Reply:
sorry, what am I missing?
![Screenshot at 2020-05-05 23-28-46](https://user-images.githubusercontent.com/39454100/81118081-87ead780-8f28-11ea-9566-163068a39414.png)

Reply:
not here (gtk 3.24.18)

@mate-desktop/core-team please test